tools/cassandra_delete_range/internal/cass/cass.go

```diff
@@ -368,70 +583,78 @@ func (c *ClioCass) prepareDeleteQueries(
 				preparedQuery := session.Query(q)
 
 				for r := range rangesChannel {
+					if c.settings.RangesRead != nil {
+						if c.settings.RangesRead.TokenRange.HasValue() {
+							if value, exists := c.settings.RangesRead.TokenRange.Value()[r.StartRange]; exists {
+								// Check for end range
+								if value == r.EndRange {
+									continue
+								}
+							}
+						} else {
+							log.Fatal("token_range should exist in continue.txt")
```

Comment:
The last run could be terminated when the first slice is processing. So we can not error out here. If the map is empty, we  just do nothing.